### PR TITLE
Added unity version to  CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by zwrawr**
In reference to issue #17.
Adds section saying that we are using unity 5.4 and that pull request have to build in 5.4 to be valid.

### CODE REVIEW

**Comment by frankitox16** [comment]
5.4? Are you using beta builds? Because I check for updates but 5.3.5f1 is the latest one for me at least...

**Comment by bjubes** [comment]
Checking for updates in unity is slightly bugged i think. Sometimes it says your on the latest version even if the website has a new one that is considered stable. Or as as stable as unity gets....

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ If you would like to contribute to this project by modifying/adding to the progr
   * Include screenshots demonstrating your change if applicable.
 7. For a video tutorial, please see: https://www.youtube.com/watch?v=-N4Cghw0l2Q
 
+# Unity Version
+We are using Unity version 5.4 .
```
Comment (WardBenjamin) [inline CONTRIBUTING.md]: Extraneous space between 5.4 and the period.

**Comment by WardBenjamin** [comment]
Fyi, 5.4.0 is the current version. Looks good

**Comment by quill18** [comment]
Groovy.

**Pull request merged**